x-pack/plugins/security/public/management/roles/roles_grid/roles_grid_page.tsx

```diff
@@ -485,6 +483,5 @@ export class RolesGridPage extends Component<Props, State> {
   }
   private onCancelDelete = () => {
     this.setState({ showDeleteConfirmation: false, selection: [] });
```

Comment:
Same as in role mappings, we'd like to keep the active selection on cancel of delete op.

```suggestion
    this.setState({ showDeleteConfirmation: false });
```